src/stan-users-guide/proportionality-constants.Rmd

```diff
@@ -0,0 +1,150 @@
+# Proportionality Constants  {#proportionality-constants.chapter}
+
+When evaluating a likelihood or prior as part of the log density computation in
+MCMC, variational inference, or optimization, it is usually only necessary to
+compute the functions up to a proportionality constant (or similarly compute
+log densities up to an additive constant). In MCMC this comes from the fact that
+the distribution being sampled does not need to be normalized (similarly
+the distribution does not need normalized to perform variational inference or
+do optimizations).
+
+There are three different syntaxes to work with distributions in Stan. The way
+to select between them is by determining if the proportionality constants are
+necessary.
```

Comment:
I wonder if it's worth mentioning the speed issue here. Because really the user shouldn't even bother thinking about using the unnormalized versions if they're not worried about the computation time, right? Plenty of models are fast enough that the user could ignore this and save themselves the time understanding the differences between the different versions. What do you think?